tests/grid/grid-test.js

```diff
@@ -93,4 +93,32 @@ describe('grid/grid', function () {
 			]);
 		});
 	});
+
+	describe('getLevelByViewport', function () {
```

Comment:
This is essential method for package. Could you add more test cases, even not so common?